src/OrbitClientProtos/capture_data.proto

```diff
@@ -155,7 +156,7 @@ message TimerInfo {
 
   int32 processor = 7;
   uint64 callstack_id = 8;
-  uint64 function_address = 9;
```

Comment:
should we rather use a new id for this?

Reply:
it doesn't break anything and actually it is rather a clarification from the service's point of view - this was always an id but for some reason it was called function_adddress.

Reply:
Right. Thanks for the clarification.